lib/std/fs/test.zig

```diff
@@ -1969,6 +1969,31 @@ test "delete a setAsCwd directory on Windows" {
     tmp.parent_dir.close();
 }
 
+test "use Lock.none on Windows" {
+    if (native_os != .windows) return error.SkipZigTest;
```

Comment:
Is there a particular reason this test is Windows-only?

Reply:
good point, the changes I made were for windows, so I didn't think to enable the test for non-windows targets. i will get on that asap.